sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -28,7 +28,7 @@
 from azure.core.tracing.decorator import distributed_trace  # type: ignore
 
 from ._cosmos_client_connection import CosmosClientConnection
-from ._base import build_options
+from ._base import build_options, stringify_auto_scale
```

Comment:
You need to either change `stringify_auto_scale` to `_stringify_auto_scale` generally, or import it here `import stringify_auto_scale as _stringify_auto_scale`. Otherwise, this leaks as public API, which we do not want. 

My recommendation would be to change the method name to have the leading underscore.

Reply:
Done!